packages/wonder-blocks-dropdown/src/components/option-item.tsx

```diff
@@ -314,7 +312,10 @@ const styles = StyleSheet.create({
     },
     itemContainer: {
         minHeight: "unset",
-        padding: spacing.xSmall_8,
+        // Make sure that the item is always at least as tall as 40px.
```

Comment:
**note:** By moving the `2px` to the container, we can ensure that the horizontal rule is fully positioned at the bottom of the cell.

Before that, we where adding a `bottom` padding of 2px after the hr.